projects/batfish/src/main/antlr4/org/batfish/grammar/flatjuniper/FlatJuniper_security.g4

```diff
@@ -626,6 +686,11 @@ seik_proposal
    )
 ;
 
+seaks_time
+:
+   DEC DASH DEC DASH DEC PERIOD variable (DASH DEC)?
```

Comment:
This rule is awkward to me. I didn't find any defined rule to parse a String like `2011-6-23.20:19:33-0700`, it will be separated into multiple tokens by `-` if I use defined `string` here.

Reply:
Just make a rule in `FlatJuniper_common.g4` like the `Cisco_common.g4` `variable_permissive` that eats up everything up to the `NEWLINE`.